srcalc/src/main/java/gov/va/med/srcalc/security/VistaUserDetailsService.java

```diff
@@ -1,56 +0,0 @@
-package gov.va.med.srcalc.security;
-
-import javax.inject.Inject;
-import javax.security.auth.login.AccountException;
-
-import gov.va.med.srcalc.domain.VistaPerson;
-import gov.va.med.srcalc.vista.VistaPersonDao;
-
-import org.springframework.security.core.userdetails.*;
-
-/**
- * A Spring UserDetailsService implementation that loads users from VistA.
- */
-public class VistaUserDetailsService implements UserDetailsService
```

Comment:
I just want to make sure I'm understanding this. The class and code to call getVistaPersonDao were removed because we are no longer authenticating on our side by using Spring security? I'm assuming the only authentication we do now is authenticating the information we receive from CPRS?